### Варіант № 3
### Завдання № 77(г)
Дано натуральне число $n$. Обчислити:
$\frac{1}{\sin1}+\frac{1}{\sin1+\sin2}+...+\frac{1}{\sin1+...+\sin n}$

In [1]:
import math

try:
    n = int(input("Enter number n: "))
    print("\n", sum(1 / sum(math.sin(j)for j in range(1, i + 1))
                for i in range(1, n + 1)))
except ValueError:
    print("Input error!")

# sum1, sum2 = 0, 0
# for i in range(1, n + 1):
#         sum1 = sum1 + math.sin(i)
#         sum2 = sum2 + 1/sum1
#     print(sum2)


 8.845740273146985


# Examples

### Example 1
Enter number n: 1
1.1883951057781212
### Example 2
Enter number n: 5
8.845740273146985
### Example 3
Enter number n: -2
0